client/galaxy/scripts/mvc/user/user-preferences.js

```diff
@@ -91,7 +91,7 @@ define( [ 'mvc/form/form-view', 'mvc/ui/ui-misc', 'utils/query-string-parsing' ]
 
     /** View of the main user preference panel with links to individual user forms */
     var View = Backbone.View.extend({
-
+        title: "User Preferences",
```

Comment:
Can we apply localization for these title texts as well just the way we do it for `menu`?

Reply:
Good idea, that should be easy to add!

Reply:
(and, done in 928ad29)

Reply:
thanks a lot :+1: